csrc/host_ir/jit.cpp

```diff
@@ -159,12 +159,18 @@ void registerExternalFunction(
       llvm::orc::ExecutorSymbolDef(addr, llvm::JITSymbolFlags::Exported);
 }
 
+// Helper function to print generated LLVM IR after each node is processed
+void printLLVMIR(llvm::Function* func, std::string_view msg) {
```

Comment:
```suggestion
void printLlvmIr(llvm::Function* func, std::string_view msg) {
```

https://google.github.io/styleguide/cppguide.html#General_Naming_Rules
> When an abbreviation is used, prefer to capitalize it as a single "word",

Reply:
I understand existing code doesn't always follow this either, but I'd like new code to do so.

Reply:
Got it, thanks!